recipe/meta.yaml

```
@@ -27,13 +27,12 @@ requirements:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
-    - boost-cpp
+    - libboost-devel
```

I think this might only actually need the headers?
```suggestion
    - libboost-headers
```